### PR TITLE
[tools/circle_plus_gen] Update flatc generated files

### PR DESCRIPTION
**Pull request by zetwhite**
This PR updates flatc generated files to sync with updated circle_traininfo.fbs.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

related : https://github.com/Samsung/ONE/issues/13099

### CODE REVIEW

File: tools/circle_plus_gen/schema/circle_traininfo_generated.py [hunk 1]
```diff
@@ -601,9 +602,38 @@ def LossReductionType(self):
             return self._tab.Get(flatbuffers.number_types.Int8Flags, o + self._tab.Pos)
         return 0
 
+    # ModelTraining
```
Comment (zetwhite) [inline tools/circle_plus_gen/schema/circle_traininfo_generated.py:605]: `circle_traininfo.fbs` is updated here - https://github.com/Samsung/ONE/pull/13000.
So, Let's update flatc generated files.

File: tools/circle_plus_gen/schema/circle_traininfo_generated.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# generated based on commit(b3ff53668e165843d99a108d16ab14bc2cfca669)
```
Comment (zetwhite) [inline tools/circle_plus_gen/schema/circle_traininfo_generated.py:1]: commit id points to here -https://github.com/Samsung/ONE/commit/b3ff53668e165843d99a108d16ab14bc2cfca669. 
It is the last commit of master branch when I created this PR.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]
LGTM

**Pull request merged**